### PR TITLE
Multiple fixes

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: octobot_trading/exchanges/exchange_builder.py [hunk 1]
```diff
@@ -82,7 +82,6 @@ async def _build_trader(self):
 
     async def _build_modes(self):
         try:
-            init_trading_mode_config(self.config)
```
Comment (Herklos) [inline octobot_trading/exchanges/exchange_builder.py:85]: Managed by tentacle manager ?

File: octobot_trading/exchanges/exchange_manager.py [hunk 2]
```diff
@@ -350,7 +350,7 @@ def _load_config_symbols_and_time_frames(self):
         client = self.exchange.client
         if client:
             self.client_symbols = client.symbols
-            self.client_time_frames = list(client.timeframes.values()) if hasattr(client, "timeframes") else []
+            self.client_time_frames = list(client.timeframes) if hasattr(client, "timeframes") else []
```
Comment (Herklos) [inline octobot_trading/exchanges/exchange_manager.py:353]: We keep only keys now right ?

Comment (GuillaumeDSM) [reply octobot_trading/exchanges/exchange_manager.py:353]: yes, list(dict) returns a list of its keys

File: octobot_trading/exchanges/exchange_builder.py (hunk 1 above)
Comment (GuillaumeDSM) [reply octobot_trading/exchanges/exchange_builder.py:85]: By OctoBot, it's related to the advanced manager (when we figure out advanced classes). Maybe we should move it into the tentacle manager if we decide it's its role to manage advanced classes in the future

Comment (Herklos) [reply octobot_trading/exchanges/exchange_builder.py:85]: Yes it should be better in Tentacle manager, I don't think Octobot should do that.

Comment (GuillaumeDSM) [reply octobot_trading/exchanges/exchange_builder.py:85]: related to https://github.com/Drakkar-Software/OctoBot-Tentacles-Manager/issues/11

**Pull request merged**